astropy/coordinates/angle_utilities.py

```diff
@@ -40,18 +40,22 @@ class _AngleParser(object):
     instead.
     """
     def __init__(self):
+        # TODO: in principle, the parser should be invalidated if we change unit
+        # system (from CDS to FITS, say).  Might want to keep a link to the
+        # unit_registry used, and regenerate the parser/lexer if it changes.
```

Comment:
Could you include a link to the discussion (issue or pr) here? I think it would be helpful if someone wants to fix this to review the previous discussion.